addons/controllers/packageinstallstatus_controller.go

```diff
@@ -297,13 +302,13 @@ func (r *PackageInstallStatusReconciler) reconcileClusterBootstrapStatus(
 
 	// we populate 'Message' with Carvel's PackageInstall 'UsefulErrorMessage' field as it contains more detailed information in case of an error
 	title := cases.Title(language.Und)
+	// skip adding current timestamp as it frequently triggers downstream controller to go through the CB resource for no reason
 	condition := clusterapiv1beta1.Condition{
 		Type: clusterapiv1beta1.ConditionType(title.String(pkgShortname)) + "-" +
 			clusterapiv1beta1.ConditionType(pkgiCondition.Type),
-		Status:             pkgiCondition.Status,
-		Message:            util.GetKappUsefulErrorMessage(pkgi.Status.UsefulErrorMessage),
-		Reason:             pkgiCondition.Reason,
-		LastTransitionTime: metav1.NewTime(time.Now().UTC().Truncate(time.Second)),
```

Comment:
It's sad to see timestamp to go, it was adding more insight to the reconciliation status. But I can imagine it adds to the churn a lot as well